lms/djangoapps/branding_by_referer/branding_by_referer.py

```diff
@@ -116,6 +143,32 @@ def process_response(self, request, response):
         return response
 
 
+    def set_cookie(self, response, cookie_value):
+        """
+        Method to set a new cookie with the passed value.
+        """
+        max_age = 30 * 24 * 60 * 60
+        expires = datetime.strftime(datetime.utcnow() + timedelta(seconds=max_age), "%a, %d-%b-%Y %H:%M:%S GMT")
+        response.set_cookie(
+            key=self.COOKIE_MARKETING_SITE_REFERER,
+            value=cookie_value,
+            max_age=max_age,
+            expires=expires,
+            domain=settings.SESSION_COOKIE_DOMAIN,
+            secure=settings.SESSION_COOKIE_SECURE or None
+        )
+
+
+    def check_feature_enable(self):
+        """
+        Check if the ENABLE_BRANDING_BY_REFERER is set and return its value.
+        """
+        features = configuration_helpers.get_value('FEATURES', None)
```

Comment:
```suggestion
        features = configuration_helpers.get_value('FEATURES', {}).get('ENABLE_BRANDING_BY_REFERER', False)
```
This way is a single line and just as resilient

Reply:
Done @felipemontoya